wp-includes/mysql/class-wp-mysql-token.php

```diff
@@ -24,6 +24,114 @@ public function get_name(): string {
 		return $name;
 	}
 
+	/**
+	 * Get the real unquoted value of the token.
+	 *
+	 * @return string The token value.
+	 */
+	public function get_value(): string {
```

Comment:
While I like that using `get_value()` is lazy and can generally nicely work for any token type where we need to interpret or normalize any values, I'm wondering how to solve the `NO_BACKSLASH_ESCAPES` SQL mode.

It's a very simple IF, but in the token instance, we just know nothing about SQL modes 🤔  The tokenizer knows it, so it could pass in some flag, or use a different token instance, but that makes it a bit less elegant.

Reply:
Could it be a constructor argument? The mode is already determined when the token is created.

Reply:
👍 Done in https://github.com/Automattic/sqlite-database-integration/pull/47/commits/6e5a8f5a5d4d373a78536e796460515ce98ec12f.